### PR TITLE
Documentation improvements

### PR DESCRIPTION
**Pull request by gabryelreyes**
Added deployment and architecture diagrams

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,18 +2,13 @@
...
-* [Installation](#installation)
-* [Overview](#overview)
-* [Usage](#usage)
-* [Commands](#commands)
-  * [Search](#search)
-* [Examples](#examples)
-* [Used Libraries](#used-libraries)
-* [Issues, Ideas And Bugs](#issues-ideas-and-bugs)
-* [License](#license)
-* [Contribution](#contribution)
```
Comment (BlueAndi) [inline README.md:16]: Why TOC removed?

File: doc/uml/architecture.puml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    component "Polarion" as polarion
+}
+
+package "Python" as python {
+    component "Argparse" as argparse
+}
+
+main *--> polarion : <<uses>>
+main o--> cmd : 0..*
+main *--> argparse : <<uses>>
```
Comment (BlueAndi) [inline doc/uml/architecture.puml]: Remove stereotype, not required for the aggregation/composition.

File: doc/uml/deployment.puml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+polarion -- svnRepo
+
+node "Continuous Integration Server"<<virtual-machine>> {
+
+    package "scripts" {
+        component "pyPolarionCli" as pyPolarionCli
+    }
+}
+
+polarionSoapApi )-- pyPolarionCli: <<uses>>
```
Comment (BlueAndi) [inline doc/uml/deployment.puml]: Remove stereotype, not required for client/server interface.

**Review by BlueAndi — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (gabryelreyes) [reply README.md:16]: TOC is integrated and generated automatically by GitHub (top right corner)

File: doc/uml/deployment.puml (hunk 3 above)
Comment (gabryelreyes) [reply doc/uml/deployment.puml]: image
It was present in the deployment of pyMetricCli. Should I remove it from it too?

File: README.md (hunk 1 above)
Comment (gabryelreyes) [reply README.md:16]: image

File: doc/uml/deployment.puml (hunk 3 above)
Comment (BlueAndi) [reply doc/uml/deployment.puml]: yep

**Review by BlueAndi — APPROVED** [review]

**Pull request merged**